content/src/main/content/jcr_root/apps/acs-tools/components/dumplibs/content.jsp

```diff
@@ -0,0 +1,25 @@
+<%--
+  ~ #%L
+  ~ ACS AEM Tools Bundle
+  ~ %%
+  ~ Copyright (C) 2015 Adobe
+  ~ %%
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~ #L%
+  --%>
+
+<%@include file="/libs/foundation/global.jsp" %>
+<div id="acs-tools-dumplibs">
```

Comment:
could the servlet context be provided via a data-* here? Or is that an anti-pattern for vue?